java/src/client/turntracker/TurnTrackerController.java

```diff
@@ -98,6 +90,8 @@ private void initFromModel() {
      */
     @Override
     public void update(Observable observable, Object o) {
```

Comment:
This should be overriding `Controller.updateFromModel`, not implementing `Observer.update()`. The base class already handles that.